elasticsearch_collectd.py

```diff
@@ -909,6 +919,26 @@ def load_es_info():
                (ES_VERSION, ES_CLUSTER, ES_MASTER_ELIGIBLE))
 
 
+def detect_es_master():
+    """Determines if this is the current master. This method sets ES_CURRENT_MASTER"""
+    global ES_CURRENT_MASTER
+
+    json = fetch_url("http://" + ES_HOST + ":" + str(ES_PORT) +
```

Comment:
We should definitely avoid doing this on the fetch path (since that adds latency). Since the node name is stable, we should retrieve that when the plugin starts and stick it in a global variable.

@charliesignalfx I think we should fix this, otherwise we are hitting the _stats endpoint twice which is a waste.